src/styles/utils/variables.scss

```diff
@@ -0,0 +1,7 @@
+$black-color: #000;
+$blue-color: #0700ff;
+$green-color: #2c8000;
+$orange-color: #f6a603;
+$watch-height: 80vmin;
+$watch-width: 80vmin;
+$duration: 10s;
```

Comment:
The `$duration` variable is set to `10s`. Verify that this duration is correct for the speed-up effect of the stopwatch's second hand animation. It should match the intended speed-up requirement from the task description.